test/tables/sortable-table.test.js

```diff
@@ -162,6 +162,33 @@ test('column can have custom row component', () => {
   expect(wrapper.find(MyRow).first().props().data).toEqual({ name: 'Kim', test: true })
 })
 
+test('column can have custom header component', () => {
+  const MyHeader = ({ column: { name } }) => <th>{ name }</th> // eslint-disable-line
+  const wrapper = mount(
+    <SortableTable data={ tableData } headerComponent={ MyHeader }>
+      <Column name="name"/>
+      <Column name="date"/>
+    </SortableTable>
+  )
+  expect(wrapper.find(MyHeader).exists()).toEqual(true)
```

Comment:
I prefer to use `toBe` to test exact equality
https://jestjs.io/docs/en/using-matchers#common-matchers